src/runtime/execute_process_service.ts

```diff
@@ -223,6 +227,30 @@ export class ExecuteProcessService implements IExecuteProcessService {
     }
   }
 
+  private _validateSingleStartEvent(processModel: Model.Types.Process): void {
+    const processModelFacade: IProcessModelFacade = new ProcessModelFacade(processModel);
+    const startEvents: Array<Model.Events.StartEvent> = processModelFacade.getStartEvents();
+
+    const multipleStartEventsDefined: boolean = startEvents.length > 1;
+    if (multipleStartEventsDefined) {
+      const startEventIds: Array<String> = startEvents.map((currentStartEvent: Model.Events.StartEvent) => {
+        return currentStartEvent.id;
+      });
+
+      const errorMessage: string = 'The Process Model contains multiple StartEvents, but no initial StartEvent was defined.';
+      const badRequestError: BadRequestError = new BadRequestError(errorMessage);
+
+      const additionalInfos: any = {
+        message: errorMessage,
```

Comment:
This message should be something like 
```md
The ProcessModel contains the following StartEvents
```

not the error message itself.

Then the final error object will look something like this:
```js
BadRequestError {
  message: 'The Process Model contains multiple StartEvents, but no initial StartEvent was defined.',
  code: 400,
  stack: <insert stack trace here>
  additionalInformation: {
    message: 'The ProcessModel contains the following StartEvents',
    startEventIds: [...],
  }
}
```

I hope this illustrates on why I'd rather have the message I proposed.

```suggestion
        message: 'The ProcessModel contains the following StartEvents',
```